guides/RELEASE-PROCESS.md

```diff
@@ -12,13 +12,33 @@
 
       **Note**: Your NPM account must access to the CMS group and provide a valid access token. To add a token, edit your `~/.npmrc` file so the contents are `//registry.npmjs.org/:_authToken={token}`
 
-   1. Run the release script to automatically begin the release process and publish to NPM.
+   1. Run the prepublish script to automatically prepare the NPM package for publishing.
+
       ```
-      yarn release
+      yarn pre-release
       ```
+
       This script will install dependencies, build design system assets, and run the linter and tests. If everything passes, `lerna` will prompt you for the new version number, which should follow the [SemVer release format](#versioning). After the versions are updated, the script will create a tagged release commit (prefaced with `core-`), a branch (prefaced with `release-`) containing the release commit, and a zip for the Github release notes.
-   1. The last step in the script will prompt you to continue with publishing to NPM. Proceed if everything is ready.
-   1. After the release script is completed, make sure to merge the release branch (i.e. `release-2.0.0`) into master.
+
+   1. After the prepublish script is completed, visually check that release zip created from the release script (i.e. `cmsgov-design-system-2.0.0.tgz`) contains the relevant folders and files
+
+      - cmsgov-design-system includes `dist`, `src` folders and `package.json`, `README.md` files
```

Comment:
I know Scott suggested this, but I personally dont think its necessary to try and specify the files/folders, because it honestly depends. There could be additional files that are added in the future that arent accounted for here (i.e. the `types` or `locales` folders in HC.gov child ds), and the files/folders listed here are always subject to change. To a certain extent, the person releasing the design system has to know what is needed in the release

Reply:
However, one thing that might be important to note is double checking that the release zips for `design-system` contain the `dist` folder

Reply:
I was thinking about the cases when we have changes, however, I think having this top level files would be helpful during the release. How about wording it as it typically would include, that way the content can change:
`The packages would typically contain:`
      - cmsgov-design-system: `dist`, `src` folders and `package.json`, `README.md` files
      - cmsgov-design-system-docs: `src` folder and `package.json`, `README.md` files
      - cmsgov-design-system-scripts: `gulp`, `jest` folders and `cli.js`, `configDefaults.js`, `package.json`, `README.md` files
`

Reply:
I personally dont think this extra information is helpful. The important stuff to check is inside `dist`, and again that depends on the design system. 

`src`, `package.json` and `README.md` will always be there, and the files in `design-system-scripts` aren't manipulated by the build scripts at all. Again, the only important files to check are inside `dist`.

All of these files are subject to change meaning this documentation is going to be quickly outdated. Considering how it makes the release process document longer and more verbose, I would prefer to remove it.